main/ertp/api/issuer.md

```diff
@@ -1,214 +1,275 @@
+const { mint: titleMint, issuer: titleIssuer, amountMath: titleAmountMath } = makeIssuerKit('alamedaCountyPropertyTitle', 'strSet');
 
 // These are merely amounts describing digital assets, not minting assets.
-const cornerProperty = amountMath.make(harden['1292826']);
-const adjacentProperty = amountMath.make(harden['1028393']);
-const combinedProperty = amountMath.make(harden['1292826', '1028393']);
+const cornerProperty = titleAmountMath.make(harden['1292826']);
+const adjacentProperty = titleAmountMath.make(harden['1028393']);
+const combinedProperty = titleAmountMath.make(harden['1292826', '1028393']);
 ```
 
-## issuer.getBrand()
-- Returns: `{Brand}` - The brand for the issuer.
+## issuer.getAllegedName()
+- Returns: `{allegedName}`
+
+Returns the `allegedName` for this `issuer`.
 
-Get the Brand for this Issuer. The Brand indicates the kind of digital asset and is shared by the mint, the issuer, and any purses and payments of this particular kind. The brand is not closely held, so this function should not be trusted to identify an issuer alone. Fake digital assets and amounts can use another issuer's brand.
+An alleged name is a human-readable string name 
+of a kind of digital asset. An alleged name
+should not be trusted as accurate since there is
+no public registry or expectation of uniqueness. This
+means there can be multiple issuers/mints/brands with the
+same alleged name, and thus the name by itself does not
+uniquely identify an issuer. Rather, the `brand` object does that.
 
+To put it another way, nothing stops different people from creating
+multiple `issuers` with the alleged name `Quatloos`...but that doesn't
+make any of them **the** Quatloos `issuer`. The alleged name is just a 
+human readable version which is helpful for debugging. 
 ```js
-const { issuer, brand } = makeIssuerKit('bucks');
-const bucksBrand = issuer.getBrand();
-// brand === bucksBrand
+const { issuer: quatloosIssuer } = makeIssuerKit('quatloos');
+const quatloosIssuerAllegedName = quatloosIssuer.getAllegedName();
+// quatloosIssuerAllegedName === 'quatloos'
 ```
 
-## issuer.getAllegedName()
-- Returns: `{allegedName}`
+## issuer.getAmountMathKind()
+- Returns: `{MathKind}`
 
-Get the `allegedName` for this mint/issuer.
+Get the kind of this `issuer`'s `amountMath`. It returns one of
+`MathKind.NAT` (`nat`), `MathKind.STR` (`str`), or `MathKind.STRING_SET` (`strSet`).
+
+The `amountMathKind` value specifies which of three kinds an `amountMath` is,
+and what kind of values it is used on. Each kind implements all of the same set 
+of API methods (i.e. `amountMath` methods are polymorphic). 
+- `MathKind.NAT` (`nat`): Used with fungible assets. `amount` values are natural numbers (non-negative integers). Default value.
+- `MathKind.STRING_SET` (`strSet`): Used with non-fungible assets. `amount` values are strings.
+- `MathKind.SET` (`set`): Used with non-fungible assets. `amount` values are objects or records with multiple properties.
```

Comment:
I don't think it's helpful to explain that `MathKind.NAT` corresponds to `'nat'` everytime MathKind is used. We defined MathKind so they don't have to be aware of the strings.  I would remove the strings almost everywhere.  (At least almost everywhere on this page.)

Reply:
The reason they're there so many times is that people usually use API references one method at a time, rather than going through from the start (random vs. sequential access : -)). So I shouldn't put something about a method just once at the top, even if I already wrote it about another method. I think the question here is remove the 'nat' etc. strings from everywhere in the API doc except maybe the top of the Amount Math section and just mention the values in the ERTP Guide, or leave them in as is. Can go either way, but wanted to be sure you knew why I had them in there before making the decision.